### PR TITLE
Msw 100127766 revise stepping algorithm

### PR DESCRIPTION
**Pull request by michaelwehrley**

### CODE REVIEW

File: app/models/phq_stepping.rb [hunk 1]
```diff
@@ -15,7 +15,24 @@
 # If PHQ-9 < 10 and >= 5, then "stay" i-CBT
 # If PHQ-9 < 5 at two consecutive weeks, schedule "post-engagement" coach call
 class PhqStepping
-  EARLIEST_ELIGIBLE_STEPPING_WEEK = 5
+  DANGER_LABEL = "danger"
+  SUCCESS_LABEL = "success"
+  WARNING_LABEL = "warning"
+  NO_SUGGESTION = "No"
+  YES_SUGGESTION = "YES"
```
Comment (ericcf) [inline app/models/phq_stepping.rb]: Funny that this is all caps but "No" is not :-)

File: app/models/phq_stepping.rb [hunk 2]
```diff
@@ -253,32 +282,40 @@ def consecutive_high_weeks?
...
-    subset_from = (@week < 9) ? 5 : 10
-    subset_from = 14 if @week >= 14
     @assessments_subset = @assessments.select do |assessment|
       assessment.week_of_assessment.to_i >= subset_from
     end
     true
   end
 
+  def subset_from
+    if @week < 9
```
Comment (ericcf) [inline app/models/phq_stepping.rb]: Maybe encode the "magic number" 9 in a variable?

**Comment by ericcf** [comment]
I'll be honest, I didn't read through all the specs ;-)

File: app/models/phq_stepping.rb (hunk 2 above)
Comment (michaelwehrley) [inline app/models/phq_stepping.rb]: How did I miss that?!  Thanks!

**Comment by ericcf** [comment]
:steam_locomotive:

**Pull request merged**